hs_core/hydroshare/resource.py

```diff
@@ -754,7 +754,7 @@ def add_resource_files(pk, *files, **kwargs):
             # if dir_name is empty it will add a trailing slash
             full_dir = os.path.join(base_dir, dir_name) if dir_name else base_dir
         if full_dir:
-            new_folders.add(full_dir)
+            new_folders.add(os.path.join(resource.file_path, full_dir))
```

Comment:
@sblack-usu Just wondering why resource.file_path is preappended to full_dir in new_folders list here. Is it for fixing a bug or for allowing overwrite unzipped files?

Reply:
@sblack-usu I got it when going down to review the new file - you just moved the join from there to here.